cmd/tap-fitter/main.go

```diff
@@ -16,6 +16,7 @@ type TapFitterTemplate struct {
 	compositePath         string
 	catalogPath           string
 	compositeSpec         *composite.Template
+	writer                io.Writer
 }
```

Comment:
IMHO this struct would be outsite the cmd hierarchy, like pkg or internal.

Reply:
We should have the `validateFlags` in this file, but then have the template and `GenerateDevFile` (note caps) here.